code/controller/SubsiteXHRController.php

```diff
@@ -51,6 +67,7 @@ public function getResponseNegotiator()
      */
     public function SubsiteList()
     {
-        return $this->renderWith('SubsiteList');
+        return $this->renderWith('Includes/SubsiteList');
```

Comment:
Please use `[ 'type' => 'Includes', 'SubsiteList']` instead of baking the variant into the path.

I would also suggest you consider namespacing templates. E.g. `templates\SilverStripe\Subsites\Controller\Includes\SubsiteXHRController_subsitelist.ss`. Then you could refer to the template using the below:

```php
return $this->renderWith(['type' => 'Includes', self::class . '_subsitelist']);
```